src/amo/sagas/index.js

```diff
@@ -12,10 +12,10 @@ import addons from 'core/sagas/addons';
 
 // Export all sagas for this app so runSaga can consume them.
 export default function* rootSaga() {
-  yield [
+  yield all([
```

Comment:
Not using `all()` causes a deprecation warning from `redux-saga`. See the release notes in: https://github.com/redux-saga/redux-saga/releases/tag/v0.15.0

I can add that link to the comment/commit if you'd like! 😄